allennlp/training/learning_rate_schedulers/slanted_triangular.py

```diff
@@ -138,7 +138,9 @@ def get_values(self):
                 self.batch_num_total_epoch_end[-1] / (len(self.batch_num_total_epoch_end) - 1)
             )
         else:
-            actual_num_steps_per_epoch = max(self.num_steps_per_epoch, self.last_batch_num_total)
+            actual_num_steps_per_epoch = max(
+                self.num_steps_per_epoch or 1, self.last_batch_num_total
```

Comment:
Why does it default to `1` here? Isn't that a problem when it happens?

Reply:
This gives the same behavior as it was before. I'm not sure if that's an issue.